src/components/RenderRouter/ListItem.tsx

```diff
@@ -1563,7 +1574,22 @@ class ListItem extends React.Component<Props, State> {
           ></VideoOverlay>
         </div>
       );
-    } else if (this.state.content.style === 'imageList')
+    }
+    else if(this.state.content.style ==='grid' && this.state.content.class === 'instagram'){ //This renders the instagram div and iterate tiles
+        const i:any = this.state.listData[0] as any;
+        return(
+          <div className="ListItem horizontal">
+            <div className="ListInstagramContainer">
+                {i?.items.map((tile:any, index:number)=>{
+                    return this.renderItemRouter(tile, index);
+                })}
+                </div>
+                <a className="ListInstagramButton" target="_blank" rel="noreferrer" href={i?.items[0]?.locationId ? `https://instagram.com/${i.items[0].locationId}` : `https://instagram.com/themeetinghouse/`}><img width={25} height={20} style={{marginRight:"3px"}} src="/static/svg/Instagram.svg"></img> Go to Instagram</a>
```

Comment:
Could use the `<LinkButton/>` component. Not necessary, but it would make things consistent across the codebase.